docs/get-started/dandelion-apis.md

```diff
@@ -28,11 +28,11 @@ These are the links to github repositories and a brief description for all of th
 - [KtorZ/ogmios-api][gh-ogmios]: use this JSON-WSP (websocket) API to gather live blockchain info directly from a cardano-node instance
 - [cardano-db-sync][gh-cardano-db-sync]/[postgrest-api][gh-postgrest]: use this REST API to perform SQL queries to gather info directly from cardano-db-sync's database 
 - [rosetta-api][gh-cardano-rosetta]: use this API to integrate your application with multiple blockchains in a simpler, faster, and more reliable way
-[gh-hasura-graphql]: https://github.com/input-output-hk/cardano-graphql
+[gh-hasura-graphql]: https://github.com/cardano-foundation/cardano-graphql
 [gh-cardano-rest]: https://github.com/input-output-hk/cardano-rest
```

Comment:
I think explorer-api and cardano-rest is not just deprecated, but completely non-functional since quite a few eras. we should potentially remove it, but maybe that's a seperate PR for updating this page - @rcmorano , in case you'd like to submit a new PR

Reply:
OK @rdlrt should we merge this then in the meantime, to get some of the more common bad / legacy primary links cleaned up immediately?

Reply:
yip, absolutely